targets/x11-common

```diff
@@ -7,6 +7,11 @@
 # forms of X11.
 
 ### Append to prepare.sh:
+
+#we need to install xinit to be sure we overwrite the path
+if release -eq tumbleweed; then
```

Comment:
anything that should happen on all opensuse releases should use the DISTROAKA check.